palace/utils/geodata.cpp

```diff
@@ -819,85 +822,29 @@ BoundingBox BoundingBoxFromPointCloud(MPI_Comm comm, std::vector<Eigen::Vector3d
 
 // Calculates a bounding ball from a point cloud, result is broadcast across all processes.
 BoundingBall BoundingBallFromPointCloud(MPI_Comm comm,
-                                        std::vector<Eigen::Vector3d> vertices)
+                                        std::vector<Eigen::Vector3d> &vertices,
```

Comment:
`const`

Reply:
Fixed in [fa13b91](https://github.com/awslabs/palace/pull/75/commits/fa13b9127936f410ec4f4ddb599c29abac62e848).